bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -318,11 +337,7 @@ public Builder enableEmulator(String host, int port) {
     }
 
     public BigtableOptions build() {
-      if (options.bulkOptions == null) {
-        int maxInflightRpcs =
-            BulkOptions.BIGTABLE_MAX_INFLIGHT_RPCS_PER_CHANNEL_DEFAULT * options.dataChannelCount;
-        options.bulkOptions = BulkOptions.builder().setMaxInflightRpcs(maxInflightRpcs).build();
-      } else if (options.bulkOptions.getMaxInflightRpcs() <= 0) {
+      if (options.bulkOptions.getMaxInflightRpcs() <= 0) {
```

Comment:
can we still add a check for null and throw if it is not set?

Reply:
Thanks for the review, Added a Precondition check for bulk options.